Public/Src/Cache/ContentStore/Distributed/Sessions/DistributedContentSession.cs

```diff
@@ -264,11 +264,20 @@ private Task<PutResult> PerformPutFileGatedOperationAsync(OperationContext opera
             if (registerResult && Settings.ProactiveCopyMode != ProactiveCopyMode.Disabled)
             {
                 // Since the rest of the operation is done asynchronously, create new context to stop cancelling operation prematurely.
-                WithOperationContext(
+                var proactiveCopyTask = WithOperationContext(
                     context,
                     CancellationToken.None,
-                    operationContext => RequestProactiveCopyIfNeededAsync(operationContext, result.ContentHash, path)

```

Comment:
>RequestProactiveCopyIfNeededAsync [](start = 40, length = 33)

In this case you're awaiting the result but ignoring the errors.

So you can trace explicitly, or do our normal:

if (!proactiveCopyResult.Success) {return ...}

Reply:
Or you can explicitely ignore the result and add a comment saying that the error is traced.

---
In reply to: [342737938](https://github.com/microsoft/BuildXL/pull/1141#discussion_r342737938) [](ancestors = 342737938)